zcash_client_sqlite/src/prepared.rs

```diff
@@ -331,6 +336,80 @@ impl<'a, P> DataConnStmtCache<'a, P> {
 }
 
 impl<'a, P: consensus::Parameters> DataConnStmtCache<'a, P> {
+    /// Inserts a sent note into the wallet database.
+    ///
+    /// `output_index` is the index within the transaction that contains the recipient output:
+    ///
+    /// - If `to` is a Sapling address, this is an index into the Sapling outputs of the
+    ///   transaction.
+    /// - If `to` is a transparent address, this is an index into the transparent outputs of
+    ///   the transaction.
+    /// - If `to` is an internal account, this is an index into the Sapling outputs of the
+    ///   transaction.
```

Comment:
This does not mention what happens when `to` is a UA, which is indicative of the bug I identified above.

Reply:
Not a bug.

Reply:
It _was_ a bug at the time, but it looks like the most recent commit fixes it.